packages/db/src/search/GranuleSearch.ts

```diff
@@ -116,9 +119,30 @@ export class GranuleSearch extends BaseSearch {
    * @param pgRecords - postgres records returned from query
    * @returns translated api records
    */
-  protected translatePostgresRecordsToApiRecords(pgRecords: GranuleRecord[])
-    : Partial<ApiGranuleRecord>[] {
+  protected async translatePostgresRecordsToApiRecords(pgRecords: GranuleRecord[], knex: Knex)
+    : Promise<Partial<ApiGranuleRecord>[]> {
     log.debug(`translatePostgresRecordsToApiRecords number of records ${pgRecords.length} `);
+
+    const cumulusIds = pgRecords.map((record) => record.cumulus_id);
+
+    //get Files
+    const fileModel = new FilePgModel();
+    const files = await fileModel.searchByGranuleCumulusIds(knex, cumulusIds);
+    const fileMapping: { [key: number]: PostgresFileRecord[] } = {};
+    files.forEach((file) => {
+      if (!(file.granule_cumulus_id in fileMapping)) {
+        fileMapping[file.granule_cumulus_id] = [];
+      }
+      fileMapping[file.granule_cumulus_id].push(file);
+    });
+
+    //get Executions
+    const executionUrls = await getExecutionInfoByGranuleCumulusIds({
```

Comment:
Might be danger here - the API spec uses executionUrls[0], and sort order is going to matter coming out of this in getting a consistent result.     The *intent* I believe is to provide the *latest* execution in update order.

Reply:
🤦🏻  ignore this, for some reason I assumed this was a new method.    Resolving.